en/mozillavpn.xlf

```diff
@@ -0,0 +1,505 @@
+      <source xml:space="preserve">We detected and fixed a serious bug.\nYou must update your app.</source>
+      <context-group purpose="location"><context context-type="linenumber">60</context></context-group>
+    </trans-unit>
+    <trans-unit id="updateConnectionInsecureWarning">
+      <source xml:space="preserve">Your connection will not be secure while you update.</source>
+      <context-group purpose="location"><context context-type="linenumber">172</context></context-group>
+    </trans-unit>
+    <trans-unit id="notNow">
+      <source xml:space="preserve">Not now</source>
+      <context-group purpose="location"><context context-type="linenumber">194</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="cpp" original="../src/connectiondataholder.cpp" source-language="en" target-language="en-US"><body>
+    <trans-unit id="unknown">
+      <source xml:space="preserve">Unknown</source>
+      <context-group purpose="location"><context context-type="linenumber">24</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="cpp" original="../src/main.cpp" source-language="en" target-language="en-US"><body>
+    <trans-unit id="startMinimized">
+      <source xml:space="preserve">Start minimized</source>
+      <context-group purpose="location"><context context-type="linenumber">49</context></context-group>
+    </trans-unit>
+    <trans-unit id="startOnBoot">
+      <source xml:space="preserve">Start at boot (if configured)</source>
+      <context-group purpose="location"><context context-type="linenumber">55</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="cpp" original="../src/systemtrayhandler.cpp" source-language="en" target-language="en-US"><body>
+    <trans-unit id="systrayQuit">
+      <source xml:space="preserve">Quit</source>
+      <context-group purpose="location"><context context-type="linenumber">14</context></context-group>
+    </trans-unit>
+    <trans-unit id="systrayStatusConnected">
+      <source xml:space="preserve">Mozilla VPN connected</source>
+      <context-group purpose="location"><context context-type="linenumber">29</context></context-group>
+    </trans-unit>
+    <trans-unit id="TODO">
+      <source xml:space="preserve" />
+      <context-group purpose="location"><context context-type="linenumber">29</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">34</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">39</context></context-group>
+    </trans-unit>
+    <trans-unit id="systrayStatusDisconnected">
+      <source xml:space="preserve">Mozilla VPN disconnected</source>
+      <context-group purpose="location"><context context-type="linenumber">34</context></context-group>
+    </trans-unit>
+    <trans-unit id="systrayStatusSwitch">
+      <source xml:space="preserve">Mozilla VPN switching</source>
```

Comment:
This needs a comment: is it "switching" as in "it's switching server"? Should it have an ellipsis?